Description: Resolve FTBFS with GCC 8
 We can't use the regular C++17 <filesystem> header, because the code is written
 against the pre-C++17 header <experimental/filesystem>.
Author: Juhani Numminen <juhaninumminen0@gmail.com>
Bug-Debian: https://bugs.debian.org/897734
Forwarded: http://bugs.thedarkmod.com/view.php?id=4874
Last-Update: 2018-05-24
--- configure.ac
+++ configure.ac
@@ -159,15 +159,6 @@
 # By default, we rely on boost.filesystem, unless we have the C++17 libs 
 use_boost_filesystem=yes
 
-# Check for the regular C++17 <filesystem> header
-AC_CHECK_HEADER([filesystem], [found_std_filesystem=yes], [found_std_filesystem=no])
-
-# if no <filesystem> is present, check for the pre-C++17 header <experimental/filesystem>
-if test "$found_std_filesystem" = "yes"
-then
-    AC_DEFINE([HAVE_STD_FILESYSTEM], [1], [Define this to indicate that the <filesystem> header is available for inclusion.])
-    use_boost_filesystem=no
-else
     AC_CHECK_HEADER([experimental/filesystem], [found_experimental_filesystem=yes], [found_experimental_filesystem=no])
     
     if test "$found_experimental_filesystem" = "yes"
@@ -176,7 +167,6 @@
                                                         header is available for inclusion.])
         use_boost_filesystem=no
     fi
-fi
 
 FILESYSTEM_LIBS=''
 


